### PR TITLE
Appveyor FCS build

### PR DESCRIPTION
**Pull request by DedSec256**

### CODE REVIEW

File: tests/service/EditorTests.fs [hunk 1]
```diff
@@ -311,7 +311,7 @@ type Test() =
     let parseResult, typeCheckResults =  parseAndCheckScript(file, input) 
 
     let decls = typeCheckResults.GetDeclarationListInfo(Some parseResult, 4, inputLines.[3], PartialLongName.Empty(20), (fun _ -> []))|> Async.RunSynchronously
-    let item = decls.Items |> Array.tryFind (fun d -> d.Name = "abc")
+    let item = decls.Items |> Seq.tryFind (fun d -> d.Name = "abc")
```
Comment (auduchinok) [inline tests/service/EditorTests.fs]: Please move it to a separate PR.

Comment (DedSec256) [reply tests/service/EditorTests.fs]: moved to https://github.com/auduchinok/visualfsharp/pull/5
